deploy/kubernetes/base/daemonset.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1beta2 # for versions before 1.8.0 use extensions/v1beta1 [apps/v1beta2]
```

Comment:
for *Kubernetes* versions ...

same below.

Reply:
done